spec/features/course/experience_points/forum_disbursement_spec.rb

```diff
@@ -28,36 +28,43 @@
                                           created_at: 3.weeks.ago, updated_at: 3.weeks.ago)
         end
         create(:course_discussion_post_vote, post: older_posts[0])
-        visit forum_disbursement_course_users_path(course)
+        visit disburse_experience_points_course_users_path(course)
 
         within find(content_tag_selector(students[0])) do
-          expect(find('.points_awarded').value).to eq('100')
+          expect(find('input').value).to eq('100')
         end
 
+        start_date = 4.weeks.ago
+        end_date = 2.weeks.ago
+        fill_in_mui_datetimepicker(find('div.start_time'), start_date)
+        fill_in_mui_datetimepicker(find('div.end_time'), end_date)
+
+        find('div.weekly_cap').find('input').set(200)
+
         within find('.forum-participation-search-panel') do
-          fill_in 'experience_points_forum_disbursement[start_time]', with: 4.weeks.ago
-          fill_in 'experience_points_forum_disbursement[end_time]', with: 2.weeks.ago
-          fill_in 'experience_points_forum_disbursement[weekly_cap]', with: 200
-          click_button I18n.t('course.experience_points.forum_disbursement.new.search')
+          find('button.filter-btn-submit').click
+          sleep 0.2
         end
 
+        # TODO: figure out why its 429 points
```

Comment:
I was doing the tests when I realised the points changed, but am not very sure how the points are calculated exactly, is it okay to just change them? I think it could be to do with the exact time, as I only fixed the year, month and date, but not the hours & seconds (was not able to select and click on the hours and seconds)

Reply:
This is fine